### PR TITLE
[OWA-89] feat: rewrite mediaURL function

### PR DESCRIPTION
**Pull request by CarinaDraganJW**
## Description

- Rewrite `mediaURL` function to accept id and title instead of the whole media item. 

### Steps completed:

According to our definition of done, I have completed the following steps:

- [ ] Acceptance criteria met
- [ ] Unit tests added
- [ ] Docs updated (including config and env variables)
- [ ] Translations added
- [ ] UX tested
- [ ] Browsers / platforms tested
- [ ] Rebased & ready to merge without conflicts
- [ ] Reviewed own code

### CODE REVIEW

**Review by langemike — APPROVED** [review]
LGTM!

File: packages/common/src/utils/urlFormatting.ts [hunk 1]
```diff
@@ -137,7 +135,7 @@ export const determinePath = ({ type, contentId }: { type: AppMenuType | undefin
     case APP_CONFIG_ITEM_TYPE.content_list:
       return contentListURL(contentId);
     case APP_CONFIG_ITEM_TYPE.media:
-      return singleMediaURL(contentId);
+      return mediaURL({ id: contentId });
```
Comment (ChristiaanScheermeijer) [inline packages/common/src/utils/urlFormatting.ts]: We could optionally use the label as title here (`Layout.tsx` -> navItems -> label)

File: packages/common/src/utils/urlFormatting.ts [hunk 2]
```diff
@@ -132,12 +130,12 @@ export const contentListURL = (id: string, title?: string) => {
...
 };
 
-export const determinePath = ({ type, contentId }: { type: AppMenuType | undefined; contentId: string }) => {
+export const determinePath = ({ type, contentId, label }: { type: AppMenuType | undefined; contentId: string; label?: string }) => {
   switch (type) {
     case APP_CONFIG_ITEM_TYPE.content_list:
       return contentListURL(contentId);
     case APP_CONFIG_ITEM_TYPE.media:
-      return singleMediaURL(contentId);
+      return mediaURL({ id: contentId, title: label });
```
Comment (ChristiaanScheermeijer) [inline packages/common/src/utils/urlFormatting.ts:138]: Last one! We can use `label` for the `contentListURL` and `playlistURL` formatter as well 😅

Comment (CarinaDraganJW) [reply packages/common/src/utils/urlFormatting.ts:138]: Nice catch, thank you @ChristiaanScheermeijer ❤️
I will fix it now 🚀

File: packages/ui-react/src/pages/ScreenRouting/mediaScreens/MediaEpisode/MediaEpisode.tsx [hunk 3]
```diff
@@ -31,7 +31,18 @@ const MediaEpisode: ScreenComponent<PlaylistItem> = ({ data: media, isLoading: i
   }
 
   // Use media episode item for legacy series flow
-  return <Navigate to={mediaURL({ episodeId: mediaId, media: seriesMedia as PlaylistItem, play, playlistId: feedId })} replace />;
+  return (
+    <Navigate
+      to={mediaURL({
+        episodeId: mediaId,
+        id: (seriesMedia as PlaylistItem)?.mediaid,
+        title: (seriesMedia as PlaylistItem)?.title,
```
Comment (AntonLantukh) [inline packages/ui-react/src/pages/ScreenRouting/mediaScreens/MediaEpisode/MediaEpisode.tsx]: I would suggest to destructure them first and then pass as props.

**Review by ChristiaanScheermeijer — APPROVED** [review]
Nice @CarinaDraganJW!

**Review by AntonLantukh — APPROVED** [review]

**Pull request merged**